aptos/token_bridge/sources/structs/transfer_result.move

```diff
@@ -1,21 +1,22 @@
 module token_bridge::transfer_result {
     use wormhole::u256::{U256};
     use wormhole::u16::{U16};
+    use wormhole::external_address::{ExternalAddress, from_vector};
 
     friend token_bridge::transfer_tokens;
 
     struct TransferResult {
         // Chain ID of the token
         token_chain: U16,
         // Address of the token. Left-zero-padded if shorter than 32 bytes
-        token_address: vector<u8>,
+        token_address: ExternalAddress,
         // Amount being transferred (big-endian uint256)
         normalized_amount: U256,
         // Amount of tokens (big-endian uint256) that the user is willing to pay as relayer fee. Must be <= Amount.
         normalized_relayer_fee: U256,
     }
 
-    public fun destroy(a: TransferResult): (U16, vector<u8>, U256, U256) {
+    public fun destroy(a: TransferResult): (U16, ExternalAddress, U256, U256) {
```

Comment:
this type is now so much nicer to look at 🙂

Reply:
agree!